### DISCUSSION TITLE
[v4] @tailwindcss/postcss returns error on windows x64

### DISCUSSION THREAD

**Discussion by Mankindofficial** [tailwindlabs/tailwindcss#15268] [Help]
Got this error while starting up my dev server after using the upgrade to v4 command

[Failed to load PostCSS config: Failed to load PostCSS config (searchPath: C:/Documents/TailwindProj): [Error] Loading PostCSS Plugin failed: Cannot find module 'C:\Documents\TailwindProj\node_modules@tailwindcss\oxide-win32-x64-msvc\tailwindcss-oxide.win32-x64-msvc.node'. Please verify that the package.json has a valid "main" entry

Any clue or advice would be very much appreciated.

**Comment by christopher356** [comment]
The error indicates a missing native module. Try reinstalling Tailwind CSS and its dependencies, checking Node.js version, verifying tailwind.config.js, and reinstalling native modules. [Login & Support](https://www.adp-run.com/)

**Comment by wongjn** [comment]
Try this solution: https://github.com/tailwindlabs/tailwindcss/discussions/15701#discussioncomment-11952654

**Comment by yash-k-s-7span** [comment]
@Mankindofficial 
- Install the **Latest Microsoft Visual C++ Redistributable Version** [https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170)
- Then restart your device and then again install tailwindcss v4.0.
